docs/src/rules/line-comment-position.md

```diff
@@ -3,6 +3,7 @@ title: line-comment-position
 rule_type: layout
 ---
 
+This rule was **deprecated** in ESLint v9.3.0.
```

Comment:
it's required by the checker. I was presuming `v9.3.0` is the upcoming version.

Reply:
Can we also add a link to the corresponding rule, like we are doing with other rules that are moved to the stylistic plugin? For example:

https://github.com/eslint/eslint/blob/a63ed722a64040d2be90f36e45f1f5060a9fe28e/docs/src/rules/array-bracket-newline.md?plain=1#L8

Reply:
It doesn't exist yet: https://github.com/eslint-stylistic/eslint-stylistic/issues/388

I can add a link when it's done.

Reply:
Ah, the rules haven't been published yet?

https://github.com/eslint-stylistic/eslint-stylistic/pull/389

In that case, merging this PR will need to wait for the new version of the plugin.

Reply:
ESLint Stylistic v2.1 has been released with these rules